### ISSUE TITLE
terminal window odd behaviour

### ISSUE THREAD

**Issue by fedef** [ntowbinj/switch-monitor#1]
Hi, thanks for the useful script.
In my PC i found out that the Terminal window has a different behaviour from the others (firefox, files..): it is only possible to switch it a random number of times between 2 monitors when it is in fullscreen.
Not a dealbreaker of course but wanted to let you know if you are interested.
I would look into that myself but i am a linux beginner.
My config:
Linux Mint 17 64 bit
Cinnamon 2.2.16
2xDell FHD(1920x1080) Monitors in "multi-display desktop with 2 displays" (from the AMD CCC).
Regards

**Comment by ntowbinj** [comment]
Are you talking about the fact that if it's not full screen and you switch it twice for instance, it's not back to the same place on the original monitor?

**Comment by fedef** [comment]
No, it happens when the Terminal application of linux mint is in fullscree. Then I can only switch it a random(to me) number of times. After that, that terminal window doesnt switch anymore.

**Comment by ntowbinj** [comment]
Hmm, I'm unable to reproduce that on my own Mint 17.  By fullscreen, do you mean legit fullscreen like when you press f11, or maximized?

**Comment by fedef** [comment]
I mean maximized
I just tried with the f11 fullscreen, because i never use it, and I dont have the problem in this case.
So it seems to be just the maximized terminal window
Btw i bound :
switch-monitor.sh 1 => super alt right
switch-monitor.sh -1 => super alt left
I assumed it was the right way to do but maybe I forgot something

**Comment by LIttleAncientForestKami** [comment]
@fedef @ntowbinj with terminal I noticed switch-monitor.sh 1 moves window back and forth.
